nanofi/include/core/cstructs.h

```diff
@@ -114,4 +126,6 @@ typedef enum FS {
   ROLLBACK
 } FailureStrategy;
 
+typedef void (processor_logic)(processor_session*, processor_context *);
```

Comment:
You have a comment, below, about accessing properties not being supported. Does that mean that processor_logic will go in tandem with processor configuration somehow?